l10n_ch_account_tags/__init__.py

```diff
@@ -0,0 +1,207 @@
+        "ch_coa_4086": "l10n_ch_account_tags.account_tag_ch_4",
+        "ch_coa_4092": "l10n_ch_account_tags.account_tag_ch_4",
+        "ch_coa_4200": "l10n_ch_account_tags.account_tag_ch_4",
+        "ch_coa_4400": "l10n_ch_account_tags.account_tag_ch_4",
+        "ch_coa_4500": "l10n_ch_account_tags.account_tag_ch_4",
+        "ch_coa_4510": "l10n_ch_account_tags.account_tag_ch_4",
+        "ch_coa_4520": "l10n_ch_account_tags.account_tag_ch_4",
+        "ch_coa_4521": "l10n_ch_account_tags.account_tag_ch_4",
+        "ch_coa_4530": "l10n_ch_account_tags.account_tag_ch_4",
+        "ch_coa_4540": "l10n_ch_account_tags.account_tag_ch_4",
+        "ch_coa_4800": "l10n_ch_account_tags.account_tag_ch_4",
+        "ch_coa_4801": "l10n_ch_account_tags.account_tag_ch_4",
+        "ch_coa_4900": "l10n_ch_account_tags.account_tag_ch_4",
+        "ch_coa_4901": "l10n_ch_account_tags.account_tag_ch_4",
+        "ch_coa_4903": "l10n_ch_account_tags.account_tag_ch_4",
+        "ch_coa_4906": "l10n_ch_account_tags.account_tag_ch_4",
+        "ch_coa_5000": "l10n_ch_account_tags.account_tag_ch_5",
+        "ch_coa_5700": "l10n_ch_account_tags.account_tag_ch_5",
+        "ch_coa_5800": "l10n_ch_account_tags.account_tag_ch_5",
+        "ch_coa_5900": "l10n_ch_account_tags.account_tag_ch_5",
+        "ch_coa_6000": "l10n_ch_account_tags.account_tag_ch_60",
+        "ch_coa_6100": "l10n_ch_account_tags.account_tag_ch_60",
+        "ch_coa_6105": "l10n_ch_account_tags.account_tag_ch_60",
+        "ch_coa_6200": "l10n_ch_account_tags.account_tag_ch_60",
+        "ch_coa_6260": "l10n_ch_account_tags.account_tag_ch_60",
+        "ch_coa_6300": "l10n_ch_account_tags.account_tag_ch_60",
+        "ch_coa_6400": "l10n_ch_account_tags.account_tag_ch_60",
+        "ch_coa_6500": "l10n_ch_account_tags.account_tag_ch_60",
+        "ch_coa_6570": "l10n_ch_account_tags.account_tag_ch_60",
+        "ch_coa_6600": "l10n_ch_account_tags.account_tag_ch_60",
+        "ch_coa_6700": "l10n_ch_account_tags.account_tag_ch_60",
+        "ch_coa_6800": "l10n_ch_account_tags.account_tag_ch_68",
+        "ch_coa_6900": "l10n_ch_account_tags.account_tag_ch_69",
+        "ch_coa_6950": "l10n_ch_account_tags.account_tag_ch_69",
+        "ch_coa_7000": "l10n_ch_account_tags.account_tag_ch_7",
+        "ch_coa_7010": "l10n_ch_account_tags.account_tag_ch_7",
+        "ch_coa_7500": "l10n_ch_account_tags.account_tag_ch_7",
+        "ch_coa_7510": "l10n_ch_account_tags.account_tag_ch_7",
+        "ch_coa_8000": "l10n_ch_account_tags.account_tag_ch_80",
+        "ch_coa_8100": "l10n_ch_account_tags.account_tag_ch_80",
+        "ch_coa_8500": "l10n_ch_account_tags.account_tag_ch_85",
+        "ch_coa_8510": "l10n_ch_account_tags.account_tag_ch_85",
+        "ch_coa_8900": "l10n_ch_account_tags.account_tag_ch_89",
+    }
+
+    # Get the current company id (can be retrieved from env.user or env.company)
+    company = env["res.company"].search([("name", "=", "CH Company")], limit=1)
```

Comment:
why not all companies that use the l10n_ch chart of account?
`CH Company` is just a demo company